pkg/cache/task.go

```diff
@@ -282,8 +290,7 @@ func (task *Task) postTaskAllocated(event *fsm.Event) {
 			zap.String("podUID", string(task.pod.UID)))
 		if task.context.apiProvider.GetAPIs().VolumeBinder != nil {
 			if err := task.context.bindPodVolumes(task.pod); err != nil {
-				errorMessage = fmt.Sprintf("bind pod volumes failed, name: %s, uid: %s, %#v",
-					task.pod.Name, task.pod.UID, err)
+				errorMessage = fmt.Sprintf("bind pod volumes failed, name: %s, %#v", task.alias, err)
```

Comment:
error does not implement GoStringer, why use `%#v`

Reply:
fixed, we can use just string msg here.